Flow.Launcher.Plugin/VisibilityChangedEventHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Flow.Launcher.Plugin
```

Comment:
Is there any reason why this is in a file of its own instead of being placed in the EventHandler.cs file?

Reply:
No reason! Though I originally thought EventHandler.cs was only for key events.

Reply:
Yeah, mostly key events so far, but I would expect to find event handlers in a file with that filename.

Reply:
Alright! Happy for it to move 😄

Reply:
All done